src/main/java/com/revature/dtos/AddressRequest.java

```diff
@@ -10,6 +10,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class AddressRequest {
+	private int id;
```

Comment:
@hectororozco98 Can you explain why this was added here? Is this for the update method in `AddressController`?